bin/reconutils.py

```diff
@@ -5,11 +5,12 @@
 import sys
 import glob
 import numpy as np
-import nibabel
+import nibabel as nbl
```

Comment:
for future, I prefer not to abbrev module names other than numpy (which is used in arith expr)